### ISSUE TITLE
[Global Procedure Request] - codeunit 7312 "Create Pick" - local procedure to global procedure

### ISSUE THREAD

**Issue by djuras2** [microsoft/ALAppExtensions#27356]
Dear Support,
Could You please modify the CreateNewWhseDoc from local procedure to global proceedure in codeunit 7312 "Create Pick"?

<<<<< Original >>>>>

```

 local procedure CreateNewWhseDoc(OldNo: Code[20]; OldSourceNo: Code[20]; OldLocationCode: Code[10]; var FirstWhseDocNo: Code[20]; var LastWhseDocNo: Code[20]; var NoOfSourceDoc: Integer; var NoOfLines: Integer; var WhseDocCreated: Boolean)

```

<<<<< Modified >>>>>

```

procedure CreateNewWhseDoc(OldNo: Code[20]; OldSourceNo: Code[20]; OldLocationCode: Code[10]; var FirstWhseDocNo: Code[20]; var LastWhseDocNo: Code[20]; var NoOfSourceDoc: Integer; var NoOfLines: Integer; var WhseDocCreated: Boolean)

```

### Additional context

After an upgrade, we have to reuse this from a custom procedure, and we can not call it due to it is a local procedure.
Internal work item: [AB#551201](https://dynamicssmb2.visualstudio.com/1fcb79e7-ab07-432a-a3c6-6cf5a88ba4a5/_workitems/edit/551201)

**Comment by lhadhazi9940** [comment]
Any updates on this?

**Comment by JakovljevicDusan** [comment]
Already implemented for #27624 . To be released in the next major (26).
